app/controllers/iiif/auth/v2/token_controller.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# API to create IIIF Authentication access tokens
+module Iiif
+  module Auth
+    module V2
+      # Creates tokens for IIIF auth v2
+      # https://iiif.io/api/auth/2.0/#access-token-service
+      class TokenController < ApplicationController
+        skip_forgery_protection
+
+        # Returns an HTML response that posts back to the parent window
+        # See {https://iiif.io/api/auth/2.0/#workflow-from-the-browser-client-perspective}
+        def create
+          params.require(:origin)
+          params.require(:messageId)
```

Comment:
Could you do this in a single line, by passing both symbols in an array? I don't think it's a big deal either way, perhaps a bit less noisy.

Reply:
Cool. I didn't know about that way.